moduleframework/helpfile_linter.py

```diff
@@ -0,0 +1,50 @@
+from __future__ import print_function
+
+import os
+from moduleframework import common
+
+HELP_MD = "help.md"
+
+
+def get_help_md_file(dir_name):
+    helpmd_file = os.path.join(os.path.abspath(dir_name), "help", HELP_MD)
```

Comment:
we should follow fedora guidelines here: https://fedoraproject.org/wiki/Container:Guidelines#Help_File

Reply:
Good to know, but this should be also documented in distgen. Nowadays, we are building `help.1` file into `./root/help.1` directory and help.md itself is on help/ directory.